src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageExtensionUtilities.cs

```diff
@@ -77,6 +78,14 @@ public static object ConvertFromHttpMessageToExpando(RpcHttp inputMessage)
             return new Tuple<string, string, CookieOptions>(cookie.Name, cookie.Value, cookieOptions);
         }
 
+        internal static void UpdateWorkerMetadata(this WorkerInitResponse initResponse, RpcWorkerConfig workerConfig)
```

Comment:
just wondering - why is this internal rather than public?

Reply:
❤️ 🏝